docs/source/visualization/interval_color_mapping.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>Not a fan of 1 letter variables. would prefer <code>response</code> over <code>r</code> for readability.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>